### ISSUE TITLE
Non-existent config key: MODEL.BACKBONE.OUT_CHANNELS

### ISSUE THREAD

**Issue by ahong007007** [facebookresearch/maskrcnn-benchmark#563]
## To Reproduce

> from maskrcnn_benchmark.config import cfg
> from predictor import COCODemo
> import cv2
> config_file = "./e2e_mask_rcnn_X_101_32x8d_FPN_1x_caffe2.yaml"
> cfg.merge_from_file(config_file)
> cfg.merge_from_list(["MODEL.DEVICE", "cpu"])
> print(cfg)
> coco_demo = COCODemo(
>     cfg,
>     min_image_size=800,
>     confidence_threshold=0.7,
> )
> def imshow(img):
>     plt.imshow(img[:, :, [2, 1, 0]])
>     plt.axis("off")
> image = cv2.imread("./00019_48a40bf59425c4cea437dabe659746b0.jpg")
> predictions = coco_demo.run_on_opencv_image(image)
> cv2.imwrite("test.jpg",predictions)

## Environment

![image](https://user-images.githubusercontent.com/22077027/54327226-5805b400-4644-11e9-896c-02bbcff11631.png)

I just load a configuration yaml file

> cfg.merge_from_file(config_file)

[environment collection script from PyTorch]
![image](https://user-images.githubusercontent.com/22077027/54327166-1e34ad80-4644-11e9-98f2-f087f465f1c3.png)

You can get the script and run it with:
```

```

 - PyTorch Version (e.g., 1.0):  torch-nightly             1.0.0.dev20190312
                                                   torchvision-nightly       0.2.1
 - OS (e.g., Linux):   Linux 2de17d7e2498 3.10.0-327
 - How you installed PyTorch (`conda`, `pip`, source):
pip install numpy torchvision_nightly
pip install torch_nightly -f https://download.pytorch.org/whl/nightly/cu90/torch_nightly.html

 - Build command you used (if compiling from source):
 - Python version:  py3.6
 - CUDA/cuDNN version:   9.0
 - GPU models and configuration:
 - Any other relevant information:

**Comment by bernhardschaefer** [comment]
I can't reproduce the bug.
Are you sure that you are using the latest config file from the master branch?
The BACKBONE.OUT_CHANNELS property was removed in a commit 22 days ago:
https://github.com/facebookresearch/maskrcnn-benchmark/commit/b23eee0cb72af70f4e4a72e73537f0884cfd1cff#diff-ab617a9bc705bd16a74589d095e5adbf

**Comment by ahong007007** [comment]
thanks.
I found the problem. It is not consistent with the version used at the time of the training and at the time of deployment.

**Comment by JiaquanYe** [comment]
> thanks.
> I found the problem. It is not consistent with the version used at the time of the training and at the time of deployment.

你好，我也遇到这种情况，请问你是怎么解决的？我是昨天还能用的，今天用demo预测就不行了，而且发现之前能用的相关程序现在都不能用了。。。

**Comment by zardadi** [comment]
I have the same issue when running maskrcnn demo. I have checked the config file, it is updated. Any solution?

**Comment by DineshRajanT** [comment]
> thanks.
> I found the problem. It is not consistent with the version used at the time of the training and at the time of deployment.

How did you solve this?

**Comment by PareshKamble** [comment]
Hello @ahong007007, 
Can you please elaborate on how did you solve the issue?
Thanking you in anticipation!
